src/gameplay_v2/cards/cardEntity.ts

```diff
@@ -0,0 +1,28 @@
+import { Round } from '../gameTypes';
+import Component from '../roles/components/component';
+
+export default class CardEntity {
+  components: Component[];
+  round: Round;
```

Comment:
Why does a card have round? 

I haven't though too much about the card just yet btw. We should have a discussion on your design to integrate it :)